src/components/swap/components/trade-input-selector.tsx

```diff
@@ -75,6 +75,7 @@ export const TradeInputSelector = (props: TradeInputSelectorProps) => {
             placeholder='0'
             _placeholder={{ color: colors.ic.gray[400] }}
             type='number'
+            onWheel={(e) => e.target instanceof HTMLElement && e.target.blur()}
```

Comment:
epic fix! I had this before too lol

should this be just prevent default? might not be enough?

Reply:
I tried both `preventDefault` and `stopPropagation`, but neither worked for me and scrolling still changes the number. I was curious if you had come across a cleaner solution or seen something in the Chakra docs...if not, we'll just proceed with this change!

Reply:
Merging this, if you know another approach we'll swap!